pages/en/network/indexing.mdx

```diff
@@ -672,29 +674,33 @@ indexer indexer actions cancel
 
 indexer indexer actions approve 1 3 5
 
-indexer indexer actions execute approve 
+indexer indexer actions execute approve
 ```
 
 Note that supported action types for allocation management have different input requirements:
-`Allocate` - allocate stake to a specific subgraph deployment
-- required action params:
+
+- `Allocate` - allocate stake to a specific subgraph deployment
+
+  - required action params:
     - deploymentID
     - amount
 
-`Unallocate` - close allocation, freeing up the stake to reallocate elsewhere
-- required action params:
+- `Unallocate` - close allocation, freeing up the stake to reallocate elsewhere
+
+  - required action params:
     - allocationID
     - deploymentID
-- optional action params:
-    - poi
-    - force (forces using the provided POI even if it doesn’t match what the graph-node provides)
+  - optional action params:
+  - poi
+  - force (forces using the provided POI even if it doesn’t match what the graph-node provides)
```

Comment:
These should be indented, right?

Reply:
Good catch! Just fixed that - thank you!